package.json

```diff
@@ -22,13 +19,27 @@
   },
   "scripts": {
     "build": "yarn workspaces run build",
-    "build:contracts": "yarn workspace @qfi/contracts run build",
-    "build:subgraph": "yarn workspace @qfi/subgraph run build",
-    "build:ui": "yarn workspace @qfi/ui run build",
-    "build:hooks": "yarn workspace @qfi/hooks run build",
+    "build:contracts": "yarn workspace @quadratic-funding/contracts run build",
+    "build:subgraph": "yarn workspace @quadratic-funding/subgraph run build",
+    "build:ui": "yarn workspace @quadratic-funding/ui run build",
+    "build:hooks": "yarn workspace @quadratic-funding/hooks run build",
+    "lint": "npm-run-all lint:*",
+    "lint:eslint": "eslint packages",
+    "lint:prettier": "prettier --check **/*.ts",
```

Comment:
Nit: remove, do not add multiple linters to packages, follow the conventions each subpackage uses as they are maintained by different developers, should just have to run lint command at root level.